gdal/frmts/pds/pds4dataset.cpp

```diff
@@ -1152,24 +1152,59 @@ void PDS4Dataset::ReadGeoreferencing(CPLXMLNode* psProduct)
                                                      "");
         bool bIsOgraphic = EQUAL(pszLatitudeType, "Planetographic");
 
+        // Updated to use new PDS4 radius names at CART 1.B.1.0. Original names 
+        // were confusing and did not follow the recommended FGDC names. Using both 
+        // "semi" and "radius" in the same keyword, which both mean half, 
+        // does not make sense. Because of that, all PDS4 radius keywords were
+        // changed.  Thus "semi_major_radius" was changed to a_axis_radius 
+        // to be more compatible with PDS3. see next 2 sections for other updates.
         double dfSemiMajor = GetLinearValue(psGeodeticModel,
-                                            "semi_major_radius");
-        // according to the spec, it seems the semi_minor_radius is
-        // considered in the equatorial plane, which is rather unusual
-        // for WKT, we want to use the polar_radius as the actual semi minor
-        // axis
+                                            "a_axis_radius");
+        // Try to grab original PDS4 radius name from previous CART 
+        // schema pre-1.B.0.0 (pre LDD version 1.9.3.0)
+        if( dfDemiMajor != nullptr )
```

Comment:
This is incorrect. Not sure how to test for not found.

Reply:
Try `CPLGetXMLNode(pseodeticModel, "a_axis_radius") != nullptr`